src/styles/pages/_survey_thanks.scss

```diff
@@ -26,7 +26,11 @@ body.survey-pagetype.thanks-page {
   }
 
   .text-info {
-    color: #0098AD; //giraffe has #17a2b8 for .text-info but Brianna says it should be darker teal for this page.
+    // TODO: Aili set this color to #0098AD on Sep 24, 2021, with the following note:
```

Comment:
I have messaged Aili.

- [ ] Update TODO with Aili's response

Reply:
Aili doesn't have any flags on this change. I will remove the comment block entirely.